src/components/Schedule/AvailabilityBar.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from "react";
+import { ProgressBar } from "./ProgressBar";
+
+interface IProps {
+  currentReservation: number;
+  maxReservation: number;
+}
+
+export const AvailabilityBar: FC<IProps> = ({
+  currentReservation,
+  maxReservation
+}) => (
+  <>
+    <div className="flex justify-between lg:pt-20 pt-8 text-12 font-lato font-bold">
+      <div>Obsadenosť</div>
+      <div className="flex content-end">
```

Comment:
this `div` doesn't need neither `flex` or `content-end` class just use `span` on Line 17 and 18 for the reservation counts.